modules/apps/commerce/commerce-product-subscription-type-web/src/main/resources/META-INF/resources/js/yearly/view.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+
+export default function ({modeExactdayOfYear, namespace}) {
+	function changeYearlyDeliveryCPSubscriptionTypeSettingsMode() {
```

Comment:
Moving a function here will make it uncallable from the outside (have a look at its closure).

Also, the function should be invoked via event listener after removing:
`onChange='<%= "event.preventDefault(); changeYearlyDeliveryCPSubscriptionTypeSettingsMode();"`